app/controllers/api/delegations_controller.rb

```diff
@@ -47,6 +52,11 @@ def create
 
         delegation.save!
       end
+      users_to_email.uniq.each do |u|
+        if u.last_email_received < u.last_sign_in_at || Time.now.utc - 259200 >= u.last_email_received 
```

Comment:
This currently fails with a 403 if you try to delegate a question to an existing user who hasn't received an email before, and I think it's because `last_email_received` isn't defined yet. This can probably be fixed by adding a check to see if `u.last_email_received` is nil -- if so, just send the email without performing the check first